addons/l10n_dk_nemhandel/wizard/account_move_send.py

```diff
@@ -0,0 +1,195 @@
+
+from odoo import api, fields, models, _
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    checkbox_send_nemhandel = fields.Boolean(
+        string='Send via Nemhandel',
+        compute='_compute_checkbox_send_nemhandel', store=True, readonly=False,
+        help='Send the invoice via Nemhandel',
+    )
+    enable_nemhandel = fields.Boolean(compute='_compute_enable_nemhandel')
+    # technical field needed for computing a warning text about the nemhandel configuration
+    nemhandel_warning = fields.Char(compute="_compute_nemhandel_warning")
+    nemhandel_mode_info = fields.Char(compute='_compute_nemhandel_mode_info')
+
+    def _get_wizard_values(self):
+        # EXTENDS 'account'
+        values = super()._get_wizard_values()
+        values['send_nemhandel'] = self.checkbox_send_nemhandel
+        return values
+
+    @api.model
+    def _get_wizard_vals_restrict_to(self, only_options):
+        # EXTENDS 'account'
+        values = super()._get_wizard_vals_restrict_to(only_options)
+        return {
+            'checkbox_send_nemhandel': False,
+            **values,
+        }
+
+    # -------------------------------------------------------------------------
+    # COMPUTE METHODS
+    # -------------------------------------------------------------------------
+
+    @api.depends('enable_nemhandel')
+    def _compute_checkbox_send_nemhandel(self):
+        for wizard in self:
+            wizard.checkbox_send_nemhandel = wizard.enable_nemhandel
+
+    @api.depends('checkbox_send_nemhandel')
+    def _compute_checkbox_ubl_cii_xml(self):
+        # extends 'account_edi_ubl_cii'
+        super()._compute_checkbox_ubl_cii_xml()
+        for wizard in self:
+            if wizard.checkbox_send_nemhandel and wizard.enable_ubl_cii_xml and not wizard.checkbox_ubl_cii_xml:
+                wizard.checkbox_ubl_cii_xml = True
+
+    @api.depends('checkbox_send_nemhandel')
+    def _compute_mail_attachments_widget(self):
+        # EXTENDS 'account' - add depends
+        super()._compute_mail_attachments_widget()
+
+    @api.depends('move_ids')
+    def _compute_nemhandel_warning(self):
+        for wizard in self:
+            invalid_partners = wizard.move_ids.partner_id.commercial_partner_id.filtered(
+                lambda partner: partner.nemhandel_verification_state != 'valid'
+            )
+            if not invalid_partners:
+                wizard.nemhandel_warning = False
+            else:
+                names = ', '.join(invalid_partners[:5].mapped('display_name'))
+                wizard.nemhandel_warning = _("The following partners are not correctly configured to receive Nemhandel documents. "
+                                        "Please check and verify their Nemhandel endpoint and the Electronic Invoicing format: "
+                                        "%s", names)
```

Comment:
indent seems weird there and i suppose 5 an arbitrary number, if there is more then 5 wouldn't it be weird ?

Reply:
Replaced this whole field to look like other warnings (with redirection action)